.gitlab-ci.yml

```diff
@@ -2,11 +2,12 @@ stages:
   - test
   - build
 
-clippy:
+clippy-and-docs:
   image: paritytech/ci-unified:bullseye-1.70.0
   stage: test
   script:
     - cargo clippy --all-features --all-targets --locked -- -D warnings
+    - cargo doc --all-features --no-deps --locked
```

Comment:
Were the errors you fixed here all found automatically? I don't think so right? I tried to put something in `<>` and it didn't complain.

Reply:
Yes, I did not do anything more than running the command above. I had to decide how to address the warning, since it was not done automatically. I think the linter looks at what's inside the `<>`. What did you try it with? If you try to undo any of the changes to the docs, you should see that specific warning popping up again.